MdePkg/Include/Register/Intel/StmApi.h

```diff
@@ -18,7 +18,6 @@
 
 #pragma pack (1)
 
-// MU_CHANGE: Added SMM_REV_ID definition, according to STM spec

```

Comment:
Is this removal needed?

Reply:
Yes, it's not cherry-picked from directly from in the first commit of the series.